test/e2e/Makefile

```diff
@@ -59,12 +59,12 @@ push-ccm-image:
 	../../hack/release.sh -p -t pr
 
 run: $(TOOLING_BINARIES) $(E2E_DATA_DIR) push-ccm-image
-	$(GINKGO) -v . -- --e2e.config="$(E2E_CONF_FILE)" --e2e.artifacts-folder="$(E2E_ARTIFACTS)" \
+	$(GINKGO) -v -failFast . -- --e2e.config="$(E2E_CONF_FILE)" --e2e.artifacts-folder="$(E2E_ARTIFACTS)" \
```

Comment:
How does target `make e2e` end up `make run` or `make dev`? I see `make e2e` is issued in e2e.sh.